docs/1-using/3-pipelines/2-statuses.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Statuses'
+---
+
+Understanding the different statuses of your pipeline is crucial for effective monitoring and management. Below are the various statuses that a pipeline can have in Conduit.
+
+## Statuses Overview
+
+| Status                  | Description                                                                 |
+|------------------------|-----------------------------------------------------------------------------|
+| **Running**            | The pipeline is actively processing data and functioning as expected.      |
+| **Stopped**            | The pipeline has been stopped by the system, or manually by a user. |
+| **Degraded**           | The pipeline is running but is experiencing issues that may affect performance. |
```

Comment:
Hmm, this isn't true actually. A degraded pipeline is a pipeline that stopped due to an error (and, depending on what we decide, the definition might also include force-stopped pipelines).

Reply:
@hariso good reminder on the force-stopped pipelines. I'll document what's current state.